src/hpc-cache/azext_hpc_cache/_validators.py

```diff
@@ -48,3 +41,17 @@ def __call__(self, parser, namespace, values, option_string=None):
         if 'target-path' in kwargs:
             junction['targetPath'] = kwargs['target-path']
         namespace.junctions.append(junction)
+
+
+def validate_storage_account_name_or_id(cmd, namespace):
+    if namespace.storage_account:
+        from msrestazure.tools import resource_id

```

Comment:
Use "from azure.mgmt.core.tools import resource_id" instead as we'll deprecate msrestazure in the future.

Reply:
Got it~ thanks